diff --git a/pyproject.toml b/pyproject.toml
index 48cd233..35a5d69 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -94,6 +94,9 @@ Documentation = "https://plaso.readthedocs.io/en/latest"
 Homepage = "https://github.com/log2timeline/plaso"
 Repository = "https://github.com/log2timeline/plaso"
 
+[tool.setuptools]
+package-dir = {"plaso" = "plaso"}
+
 [tool.setuptools.package-data]
 plaso = [
     "data/*.*",
@@ -105,8 +108,3 @@ plaso = [
     "parsers/winreg_plugins/*.yaml",
     "preprocessors/*.yaml",
 ]
-
-[tool.setuptools.packages.find]
-where = ["."]
-include = ["plaso"]
-exclude = ["docs", "tests", "tests.*", "utils"]
